vhdbuilder/packer/components.json

```diff
@@ -363,16 +363,20 @@
       "downloadURL": "https://acs-mirror.azureedge.net/azure-cni/v*/binaries",
       "versions": [
         "1.4.43.1",
```

Comment:
Can we remove 1.4.43.1?

Reply:
Asking to avoid hitting VHD space limits

Reply:
not yet please, I will remove it when we roll the new one to AKS RP and it stops being used